modules/misc/xfconf.nix

```
@@ -0,0 +1,79 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.xfconf;
+
+  withType = v:
+    if builtins.isBool v then
+      "-t bool -s ${if v then "true" else "false"}"
+    else if builtins.isInt v then
+      "-t int -s ${toString v}"
+    else if builtins.isFloat v then
+      "-t float -s ${toString v}"
+    else if builtins.isString v then
+      ''-t string -s "${toString v}"''
+    else if builtins.isList v then
+      "-a ${concatMapStringsSep " " withType v}"
+    else
+      throw "unexpected xfconf type: ${builtins.typeOf v}";
```

Together with the `escapeShellArgs` change below:

```suggestion
    if builtins.isBool v then
      [ "-t" "bool" "-s" (if v then "true" else "false") ]
    else if builtins.isInt v then
      [ "-t" "int" "-s" (toString v) ]
    else if builtins.isFloat v then
      [ "-t" "float" "-s" (toString v) ]
    else if builtins.isString v then
      [ "-t" "string" "-s" v ]
    else if builtins.isList v then
      [ "-a" ] ++ withType v
    else
      throw "unexpected xfconf type: ${builtins.typeOf v}";
```